src/loaders/external_jpg/tvgJpgLoader.cpp

```diff
@@ -149,16 +148,32 @@ bool JpgLoader::close()
 }
 
 
-unique_ptr<Surface> JpgLoader::bitmap()
+unique_ptr<Surface> JpgLoader::bitmap(int32_t colorSpace)
 {
     if (!image) return nullptr;
+    if (colorSpace != -1 && static_cast<uint32_t>(colorSpace) != this->colorSpace) {
```

Comment:
probably you can make a separate converter that returns converted value. In this case, you can call it immediately when it's necessary, no need an additional member variable.
```
uint32_t convert(uint32_t colorSpace)
{
   uint32 tjpfColorSpace;
    ...
   return tjpfColorSpace;
}
```

Reply:
thank you for your advice. i added convert function.